great_expectations/expectations/core/expect_select_column_values_to_be_unique_within_record.py

```diff
@@ -131,13 +131,15 @@ def _atomic_prescriptive_template(
         )
 
         template_str = f"Values must always be unique across columns{mostly_str}: "
-        for idx in range(len(params["column_list"]) - 1):
-            template_str += "$column_list_" + str(idx) + ", "
-            params["column_list_" + str(idx)] = params["column_list"][idx]
-
-        last_idx = len(params["column_list"]) - 1
-        template_str += "$column_list_" + str(last_idx)
-        params["column_list_" + str(last_idx)] = params["column_list"][last_idx]
+        column_list = params.get("column_list") if params.get("column_list") else []
+        if len(column_list) > 0:
+            for idx in range(len(column_list) - 1):
+                template_str += "$column_list_" + str(idx) + ", "
+                params["column_list_" + str(idx)] = column_list[idx]
+
+            last_idx = len(column_list) - 1
+            template_str += "$column_list_" + str(last_idx)
+            params["column_list_" + str(last_idx)] = column_list[last_idx]
```

Comment:
Similar comments as before. Whatever you apply there, please also apply here!

Reply:
Will adjust